modules/ocf_admin/templates/hitch.conf.erb

```diff
@@ -0,0 +1,9 @@
+frontend = {
+	host = "*"
+	port = "6378"
+}
+backend = "[127.0.0.1]:6379"
+workers = <%= @processors['count'] %>
+keepalive = 32767
```

Comment:
Maybe add a comment about this being the maximum keepalive so nobody tries to increase it?

Reply:
Oh yeah, I forgot about that. Excellent idea.